guides/cheatsheets/git.md

```diff
@@ -19,265 +23,256 @@ Version control is the essence of engineering here at the Shed. Git is a free ve
+With platform specific installers for Git, GitHub also provides the ease of
+staying up-to-date with the latest releases of the command line tool while
+providing a graphical user interface for day-to-day interaction, review, and
+repository synchronisation.
 
 You can download Git for Windows and Mac [here](https://desktop.github.com/).
 
 ---
+
 ## **SETUP**
-----
+
+---
+
 Configuring user information used across all local repositories.
 
-*Using the command below, you can set a name that is identifiable you when review version history*.
+_Set a name that is identifiable you when review version history_.
 
-```  
-git config --global user.name “[firstname lastname]”
-```
+    git config --global user.name “[firstname lastname]”
 
-*Set an email address*.
+_Set an email address_.
 
-```
-git config --global user.email “[valid-email]”
-```
+    git config --global user.email “[valid-email]”
 
 ## **SETUP & INIT**
 
 Below are commands for initializing and cloning repositories.
 
-*Initialise an existing directory as a Git repository*.
-```
-git init
-```
-*Retrieve an entire repository from a hosted location via URL*.
-```
-git clone [url]
-```
-----
+_Initialise an existing directory as a Git repository_.
+
+    git init
+
+_Retrieve an entire repository from a hosted location via URL_.
+
+    git clone [url]
+
+---
+
 ## **STAGE & SNAPSHOT**
+
 ---
+
 Working with snapshots and the Git staging area.
 
-*Show modified files in working directory, staged for your next commit*.
-```
-git status
-```
-*Add a file as it looks now to your next commit (stage)*.
-```
-git add [file]
-```
-
-*Add everything your next commit*.
-```
-git add *
-```
-*Commit your staged content as a new commit snapshot*.
-```
-git commit -m “[descriptive message]”
-```
-
-*Upload any changes to your local repository to your remote repository*.
-```
-git push
-```
-
-*Upload any changes to your local repository to your remote repository, rewriting the git history (index)*.
-```
-git push -f   !---- USE WITH CAUTION ----!
-```
-
-
-*Unstage a file while retaining the changes in working directory*.
-```
-git reset [file]
-```
-*Diff of what is changed but not staged*.
-```
-git diff
-```
-
-*Diff of what is staged but not yet commited*.
-```
-git diff --staged
-```
-
-
-
-----
+_Show modified files in working directory, staged for your next commit_.
+
+    git status
+
+_Add a file as it looks now to your next commit (stage)_.
+
+    git add [file]
+
+_Add everything your next commit_.
+
+    git add *
+
+_Commit your staged content as a new commit snapshot_.
+
+    git commit -m “[descriptive message]”
+
+_Upload any changes to your local repository to your remote repository_.
+
+    git push
+
+_Upload any changes to your local repository to your remote repository,
+rewriting the git history (index)_.
+
+    git push -f
```

Comment:
Is it worth calling out that this is a force push and that's it's normally discouraged except for certain use cases like Squashing commits?

Further, you can push multiple branches and only force one like so:

`git push origin +issue/1 issue/2`

Where + denotes that the following branch will be force pushed (issue/1).

Reply:
Noted/Completed ☑️